messaging/src/main/java/org/axonframework/deadline/quartz/DeadlineJob.java

```diff
@@ -290,7 +290,9 @@ private static Object deserializeDeadlinePayload(Serializer serializer, JobDataM
         }
 
         private static Instant retrieveDeadlineTimestamp(JobDataMap jobDataMap) {
-            return Instant.ofEpochMilli((long) jobDataMap.get(MESSAGE_TIMESTAMP));
+            Object timestamp = jobDataMap.get(MESSAGE_TIMESTAMP);
+            if (timestamp instanceof String) return Instant.parse(timestamp.toString());
```

Comment:
We do not use single line if-statements throughout the project. Yes it's a nitty point, but I'd prefer you change it to curly braces.

Reply:
fixed in subsequent commit